### DISCUSSION TITLE
Announcement Bar full screen small bug

### DISCUSSION THREAD

**Discussion by stef41** [hbstack/.github#56] [Issues and bugs]
![Screenshot from 2024-05-14 01-55-44](https://github.com/hbstack/.github/assets/87280280/a5ed50bf-899f-42b7-8749-106a88a45429)
Hello it seem that a new line is missing after the announcement bar when the website is in big screen mode

**Comment by razonyang** [comment]
Do you have a live site to test?

> in big screen mode

How wide is the screen?

**Comment by stef41** [comment]
window.innerWidth > 990 it's just on a website locally

**Reply by razonyang** [reply]
Could not reproduce.

https://github.com/hbstack/.github/assets/17720932/d161b8b5-0efd-4d0c-ba28-3c4f922fcac8

1. What version of the module do you use?
2. Have you try upgrading modules via `hugo mod get -u`?
3. What is the OS, browser (version) and other env info?

**Comment by stef41** [comment]
Upgrading solved everything thanks!
